moabb/utils.py

```diff
@@ -19,3 +20,15 @@ def set_log_level(level="INFO"):
         level=level,
         format="%(asctime)s %(levelname)s %(threadName)s %(name)s %(message)s",
     )
+
+
+def set_download_dir(path):
```

Comment:
If you create a new function, you will need to create a test, to verify that everything is working well.

I could help you, MOABB uses [unitttest](https://docs.python.org/3/library/unittest.html) which is powerful but a little bit complicated.